schema/model.py

```diff
@@ -497,6 +497,12 @@ class PyPIOptions(StrictBaseModel):
         description="Paths to directory containing",
         examples=[["https://pypi.org/simple"]],
     )
+    no_build_isolation: list[PyPIPackageName] = Field(
```

Comment:
Can you add it to the tests for the schema. (`examples/valid/full.toml`)

Reply:
Done.